outputs.tf

```diff
@@ -26,6 +26,10 @@ output "alb" {
   value = aws_alb.alb
 }
 
+output "alb_target_group" {
+  value = aws_alb_target_group
```

Comment:
This needs to include the resource identifier, not just the resource class. It should be `aws_alb_target_group.something`.

Reply:
Look in main.tf to figure out what the `something` is.